src/BBT.StructureTools.Tests/TestTools/PopPropValueExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace BBT.StructureTools.Tests.TestTools
+{
+    using System;
+    using System.Linq.Expressions;
+    using System.Reflection;
+    using BBT.StructureTools.Extension;
+
+    /// <summary>
+    /// Extensions to generate property values
+    /// for test classes.
+    /// </summary>
+    public static class PopPropValueExtensions
+    {
+        public static void CreateValueForProperty<T>(this T target, Expression<Func<T, string>> propex)
```

Comment:
Only support string as propertytype. Remark it as comment or rename the method.
(Create a value for int is not possible)